common/customproperties.go

```diff
@@ -47,8 +47,16 @@ func (c CustomProperties) GetPropertyByName(name string) CustomProperty {
 }
 
 func (c CustomProperties) getNewProperty(name string) *custom_properties.CT_Property {
+	list := c.x.Property
+	maxPid := int32(1)
+	for _, p := range list {
+		if p.PidAttr > maxPid {
+			maxPid = p.PidAttr
+		}
+	}
 	newProperty := custom_properties.NewCT_Property()
 	newProperty.NameAttr = &name
+	newProperty.PidAttr = maxPid + 1
```

Comment:
what value should this start at?  If there is no PidAttr, then maxPid is 1 and this is set to 2?  Should those start at 1 or 0?

Reply:
this should start exactly at 2, otherwise it shows errors